docker-compose.yaml

```diff
@@ -7,23 +7,23 @@ services:
     volumes:
       - ./tests/mosquitto/mosquitto.conf:/mqtt/config/mosquitto.conf
     ports:
-      - "1883"
-      - "9001"
+      - 1883:1883
+      - 9001:9001
     command: ["mosquitto", "-c", "/mqtt/config/mosquitto.conf"]
     restart: always
     logging:
       options:
         max-size: 10m
 
-  pymodbus-server:
-    container_name: pymodbus-server
+  pymodbus:
+    container_name: pymodbus
     image: ghcr.io/pymodbus-dev/pymodbus:v3.1.0
     command: ["pymodbus.server", "--host", "0.0.0.0", "--web-port", "8080", "--verbose",  "--no-repl", "run", "--modbus-server", "tcp", "--framer", "socket", "--unit-id", "1", "--unit-id", "2", "--unit-id", "3", "--unit-id", "4", "--random", "0", "--modbus-port", "5020", "--random", "1", "--modbus-config", "/pymodbus/config.json"]
```

Comment:
Are the repeated `random` args here intentional? ` "--random", "0", ... "--random", "1"`

Reply:
I just fixed that in a different PR. But the answer here is that is an accident. We should only be using one. For the purpose of this test, we don't need random